### DISCUSSION TITLE
theme toggle now outside of collapsed menu on small screens (in Quarto 1.4)

### DISCUSSION THREAD

**Discussion by timothoms** [quarto-dev/quarto-cli#8813] [Q&A] [answered]
### Description

The position of the dark/light theme toggle in relation to the navbar menu has changed in the move from Quarto 1.3 to 1.4. As a result, the theme toggle is now placed outside of the "hamburger" menu (compare the screen shots below, [repo here](https://github.com/tjetdata/sandbox)).

Is there a way to include the theme toggle inside the collapsed "hamburger" menu, the way it was in Quarto 1.3? 

![large-screen](https://github.com/quarto-dev/quarto-cli/assets/20423161/75182eac-749e-4d5a-8a81-d359600b3dc4)
![small-screen](https://github.com/quarto-dev/quarto-cli/assets/20423161/5e0d19d5-bd65-454f-9aa1-28e346a28e47)

**Comment by mcanouil** [comment]
@dragonstyle Was it by design?

**Reply by dragonstyle** [reply]
I think it may not have been (we have one other report of an issue with the layout of the right side of the navbar). I'll open an issue and have a look.

**Reply by mcanouil** [reply]
Closing in favour of:
- https://github.com/quarto-dev/quarto-cli/issues/8830

**Reply by dragonstyle** [reply, marked as answer]
Ok I looked more closely and this is by design in Quarto 1.4. From the linked bug, here was my reply:

I've reviewed the code and this is by design in Quarto 1.4. We currently place any `tools` in the top right of the navbar when the navigation collapses rather than placing them in the menu. We felt that this accomplished a couple of things:

1) It improved discovery for tools which otherwise wouldn't be discovered since they typically appear at the bottom of the collapsed menu.

2) The appears was improved, since otherwise tools appear in 'a pile' at the bottom of the collapsed menu, which wasn't ideal.

3) We felt it matched common mobile experiences which had a `hamburger` button for showing the menu one side of the toolbar, which other global tools present on the opposite side of the toolbar.

I hope this explains our thinking a bit - if you believe this should be changed, it would helpful to hear your thinking and we can think about making an enhancement...
